docs/guides/pagination/offset-based.md

```diff
@@ -0,0 +1,394 @@
+---
```

Comment:
I think we could introduce cursor based pagination with relay spec directly at this point. The example feels too simplistic and not usable in a real context. 
If we introduced relay connections here, and the ability to efficiently paginate over object relationships the advantages of offset-based become more obvious and one can understand why graphql would recommend it

Reply:
Sure! I'll try doing this!

Reply:
If I'm not wrong you want to remove the cursor-based pagination example we have and replace it with one involving relay connections? Then I think we should add it in the cursor-based guide, not here🤔

Reply:
Yes, that's exactly what i meant. And i would explicitly name Relay, as it wil be better suited for SEO when they look for relay strawberry